conans/client/migrations_settings.py

```diff
@@ -1,1944 +1,4 @@
-                  "6", "6.1", "6.2", "6.3", "6.4", "6.5",
-                  "7", "7.1", "7.2", "7.3", "7.4", "7.5",
-                  "8", "8.1", "8.2", "8.3", "8.4",
-                  "9", "9.1", "9.2", "9.3",
-                  "10", "10.1"]
-        libcxx: [libstdc++, libstdc++11]
-        threads: [None, posix, win32] #  Windows MinGW
-        exception: [None, dwarf2, sjlj, seh] # Windows MinGW
-        cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20]
-    Visual Studio: &visual_studio
-        runtime: [MD, MT, MTd, MDd]
-        version: ["8", "9", "10", "11", "12", "14", "15", "16"]
-        toolset: [None, v90, v100, v110, v110_xp, v120, v120_xp,
-                  v140, v140_xp, v140_clang_c2, LLVM-vs2012, LLVM-vs2012_xp,
-                  LLVM-vs2013, LLVM-vs2013_xp, LLVM-vs2014, LLVM-vs2014_xp,
-                  LLVM-vs2017, LLVM-vs2017_xp, v141, v141_xp, v141_clang_c2, v142,
-                  llvm, ClangCL]
-        cppstd: [None, 14, 17, 20]
-    msvc:
-        version: ["19.0",
-                  "19.1", "19.10", "19.11", "19.12", "19.13", "19.14", "19.15", "19.16",
-                  "19.2", "19.20", "19.21", "19.22", "19.23", "19.24", "19.25", "19.26", "19.27", "19.28"]
-        runtime: [static, dynamic]
-        runtime_type: [Debug, Release]
-        cppstd: [14, 17, 20]
-    clang:
-        version: ["3.3", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "4.0",
-                  "5.0", "6.0", "7.0", "7.1",
-                  "8", "9", "10", "11"]
-        libcxx: [None, libstdc++, libstdc++11, libc++, c++_shared, c++_static]
-        cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20]
-        runtime: [None, MD, MT, MTd, MDd]
-    apple-clang: &apple_clang
-        version: ["5.0", "5.1", "6.0", "6.1", "7.0", "7.3", "8.0", "8.1", "9.0", "9.1", "10.0", "11.0", "12.0"]
-        libcxx: [libstdc++, libc++]
-        cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20]
-    intel:
-        version: ["11", "12", "13", "14", "15", "16", "17", "18", "19", "19.1"]
-        base:
-            gcc:
-                <<: *gcc
-                threads: [None]
-                exception: [None]
-            Visual Studio:
-                <<: *visual_studio
-            apple-clang:
-                <<: *apple_clang
-    qcc:
-        version: ["4.4", "5.4", "8.3"]
-        libcxx: [cxx, gpp, cpp, cpp-ne, accp, acpp-ne, ecpp, ecpp-ne]
-        cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17]
-    mcst-lcc:
-        version: ["1.19", "1.20", "1.21", "1.22", "1.23", "1.24", "1.25"]
-        base:
-            gcc:
-                <<: *gcc
-                threads: [None]
-                exceptions: [None]
-
-build_type: [None, Debug, Release, RelWithDebInfo, MinSizeRel]
-
-
-cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20]  # Deprecated, use compiler.cppstd
-"""
-
-settings_1_33_1 = """
+settings_2_0_0 = """
```

Comment:
This will also be removed in the next PR.

Reply:
Yes. I just kept to be a good way when comparing the old migrations settings. If we put all on same PR it will be easier for review.